administration/intro.adoc

```diff
@@ -4,6 +4,8 @@ Installation
 KubeVirt is a virtualization add-on to Kubernetes and this guide assumes
 that a Kubernetes cluster is already installed.
 
+If installed on OpenShift, the web console is extended for management of virtual machines.
```

Comment:
Should not we use the term OKD instead of openshift?

Reply:
Good point. If so, other occurrences need to be updated as well.